pkg/consensus/consensus.go

```diff
@@ -268,7 +268,9 @@ func (s *service) Name() string {
 func (s *service) Start() {
 	if s.started.CAS(false, true) {
 		s.log.Info("starting consensus service")
-		s.dbft.Start()
+		b, _ := s.Chain.GetBlock(s.Chain.CurrentBlockHash()) // Can't fail, we have some current block!
+		s.lastTimestamp = b.Timestamp * uint64(time.Millisecond)
+		s.dbft.Start(s.lastTimestamp)
```

Comment:
And here we need to adjust `s.lastTimestamp` with `nsInMs` constant.

Reply:
Fixed.